source/main.cpp

```diff
@@ -22,6 +39,32 @@ void handleDisplay()
     uBit.display.print(IMAGE_HAPPY);
 }
 
+void setMotorPower(char motor) {
+    char buffer[3];
+    ManagedString msg = uart->readUntil(ManagedString("\n"));
+
+    buffer[0] = SET_MOTOR_POWER;
+    buffer[1] = motor;
+    buffer[2] = atoi(msg.toCharArray());
+
+    uBit.i2c.write(I2C_ADDR, buffer, 3);
+}
+
+void handleRightMotor()
+{
+    setMotorPower(0x01);

```

Comment:
We don't need `release_fiber()` at the end of all ofthese like `main` does, right? I'm guessing not, since it's working fine.

Reply:
I looked into that. It wasn't used in any of the examples, so it doesn't appear to be needed